src/main/java/com/cegeka/academy/service/EventService.java

```diff
@@ -23,4 +25,16 @@
         return eventRepository.findAllByOwner(owner);
     }
 
+    public String updateOwnerEvent(EventDTO event){
```

Comment:
Nu ar trebui sa intprci un mesaj text din servis.
Mai de graba un cod de eroare sau un boolean si interpretezi acest raspuns in controller.